src/main/resources/application-s3.yml

```
@@ -8,4 +8,8 @@ cloud:
     region:
       static: ap-northeast-2
     stack:
-      auto: false
\ No newline at end of file
+      auto: false
+
+app:
+  server:
+    uri: ${IMAGE_PREFIX}
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)